modules/ipi-install-preparing-the-bare-metal-node.adoc

```diff
@@ -44,7 +44,7 @@ $ echo -ne "root" | base64
 $ echo -ne "password" | base64
 ----
 
-. Create a configuration file for the bare metal node.
+. Create a configuration file for the bare metal node, using the following example `bmh.yaml` file:
 +
```

Comment:
Remove this command altogether, it isn't required:
```
+
[source,terminal]
----
$ vim bmh.yaml
----
```

Reply:
Also, the callouts are quite repetitive and not very useful IMO. most of the callouts are self-explanatory. You could do something like: 

```suggestion
Create a configuration file for the bare metal node, using the following example `bmh.yaml` file, replacing values to match your environment:
```